src/osd/osd_types.h

```diff
@@ -2962,6 +2962,11 @@ struct pg_log_t {
   mempool::osd::list<pg_log_entry_t> log;  // the actual log.
   
   pg_log_t() = default;
+  pg_log_t(const pg_log_t& other) = default;
+  pg_log_t& operator=(const pg_log_t& other) = default;
+  pg_log_t(pg_log_t&& other) = default;
+  pg_log_t& operator=(pg_log_t&& other) = default;
+
```

Comment:
@cbodley thanks for reminding me. i must have missed something when experimenting my minimal reproducer. yes, the default move methods are created. n3337 §12.8.9 also confirms this.

> If the definition of a class X does not explicitly declare a move constructor, one will be implicitly declared as defaulted if and only if
> — X does not have a user-declared copy constructor,
> — X does not have a user-declared copy assignment operator,
> — X does not have a user-declared move assignment operator,
> — X does not have a user-declared destructor, and
> — the move constructor would not be implicitly defined as deleted.